src/labels/CustomLabels.labels

```diff
@@ -329,18 +665,26 @@
         <value>Setting for the Account model. &quot;Administrative&quot; is the recommended model.</value>
     </labels>
     <labels>
+        <fullName>stgHelpAccoutsDeletedIfChildContactsDeleted</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>stgHelpAccoutsDeletedIfChildContactsDeleted</shortDescription>
+        <value>Record Types of those Accounts that will be automatically deleted if all their children Contacts have been deleted.</value>
+    </labels>
+    <labels>
         <fullName>stgHelpAddressAccRecType</fullName>
         <categories>Settings</categories>
         <language>en_US</language>
-        <protected>false</protected>
+        <protected>true</protected>
```

Comment:
I am curious, why are the labels being changed to protected?

Reply:
@davidjray 
Basically I just manually changed them in the xml file. Looks like I accidentally changed existing custom labels as well. Should I change them back?

Reply:
I'm not sure. I think we should have a standard we follow. @ceiroa what are your thoughts?